specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/examples/ApprovePrivateEndpointConnection.json

```diff
@@ -0,0 +1,54 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2020-12-01",
+    "diskAccessName": "myDiskAccess",
+    "privateEndpointConnectionName": "myPrivateEndpointConnection",
+    "privateEndpointConnection": {
+      "properties": {
+        "privateLinkServiceConnectionState": {
+          "status": "Approved",
+          "description": "Approving myPrivateEndpointConnection"
+        }
+      }
+    }
+  },
+  "responses": {
+    "202": {
```

Comment:
>   "202" [](start = 1, length = 8)

It should return 201 if the initial async call returns the resource with provisioning state. Please follow the RPC.
[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put)

Reply:
@vikramd-ms

Reply:
The requirement to return 201 from PUT is a recent change to the protocol (it was checked in 17 months ago); the original RPC allowed either 201 or 202 to be returned at resource creation (this can still be seen parts of the addendum which should be clarified).
Changing from 202 to 201 requires an API version change; we'll take it as a work item, but I think it's outside the scope of this checkin